### PR TITLE
Feature/add picture flynn issue17

### PR DESCRIPTION
**Pull request by Eltom777**
Code to upload files to firebase's cloud storage. In addition, I created an object called db where you can access stuff from content on the firestore database.

### CODE REVIEW

File: myfirst.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  const os = require('os');
+  const fs = require('fs');
+
+  const busboy = new Busboy({headers: req.headers});
+  
+  let imageToBeUploaded = {};
+  let imageFileName;
+  
+  busboy.on('file', (fieldname, file, filename, encoding, mimetype) => {
+
```
Comment (Eltom777) [inline myfirst.js]: Require a here to check mimetype and ensure user only input jpeg and png files

File: myfirst.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  let imageToBeUploaded = {};
+  let imageFileName;
+  
+  busboy.on('file', (fieldname, file, filename, encoding, mimetype) => {
+      
+      const imageExtension = filename.split('.')[filename.split('.').length - 1];
+      
+      imageFileName = `${Math.round(Math.random() * 1000000000000).toString()}.${imageExtension}`;
+
+      const filepath = path.join(os.tmpdir(), imageFileName);
```
Comment (Eltom777) [inline myfirst.js]: Need to check file size being uploaded. Require a threshold for size of picture being updated

File: myfirst.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          metadata: {
+              contentType: imageToBeUploaded.mimetype
+          }
+          }
+      })
+      .then(() => {
+          const imageUrl = `https://firebasestorage.googleapis.com/v0/b/${
+          config.storageBucket
+          }/o/${imageFileName}?alt=media`;
+          return db.doc(`/users/t-flynn`).update({ imageUrl });
```
Comment (Eltom777) [inline myfirst.js]: Look here @digathomas: Accessing firestore's document using my variable db. More info on db, check the file functions/util/admin.js to see how it is created. I'm using firebase's admin-sdk to access firestore.

File: myfirst.js (hunk 1 above)
Comment (Eltom777) [reply myfirst.js]: added an if statement to check mimetype on line 24 of addPhoto.js

**Review by digathomas — APPROVED** [review]
Demoed on video call and shared screen, works perfect!

**Comment by a-iacampo** [comment]
Request no longer needed

**Pull request closed without merging**